server/src/main/java/org/apache/druid/metadata/SqlMetadataSegments.java

```diff
@@ -75,9 +76,9 @@
 /**
  */
 @ManageLifecycle
-public class SQLMetadataSegmentManager implements MetadataSegmentManager
+public class SqlMetadataSegments implements MetadataSegments
```

Comment:
Could you explain the factory association? I don't see a link.

To me, "Storage" doesn't contribute to meaning, as well as "Manager" which I removed.

`SegmentsSqlMetadata` may be a better name than `SqlMetadataSegments` though.

Reply:
Renamed the interface to `SegmentsMetadata` and the implementation class to `SqlSegmentsMetadata` respectively.

Reply:
I would like to see classes that contain DB API calls - to identify themselves in naming the consistent way across the project. So do you propose to start replacing `Manager` to `Sql` or it would be a single stray class like that?

Reply:
I don't replace `Manager` with `Sql` - `SQL` was already in the name. I just removed `Manager`.

Reply:
Got it. But what about `I would like to see classes that contain DB API calls - to identify themselves in naming the consistent way across the project. `

Reply:
I think `Sql` is that identifier.